edxval/tests/test_views.py

```diff
@@ -498,7 +498,7 @@ def test_post_videos(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(
             response.data.get("edx_video_id")[0],
-            "This field must be unique."
+            u'video with this edx video id already exists.'
```

Comment:
This test raises a DRF `ValidationError` and the message associated with that exception was changed to let the Django-generated message be returned. So I changed the test to check for the new more-specific message.